tidy3d/components/autograd/__init__.py

```diff
@@ -0,0 +1,25 @@
+from .types import (
+    AutogradFieldMap,
+    AutogradTraced,
+    TracedCoordinate,
+    TracedFloat,
+    TracedSize,
+    TracedSize1D,
+    TracedVertices,
+)
+from .utils import get_static
+
+# from .derivative_utils import DerivativeInfo, integrate_within_bounds
```

Comment:
Any reason why this is commented out?

Reply:
Because it causes circular imports when this directory is imported into eg base.py. I'll remove the comment though

Reply:
Actually i need to double check this. It might be fixed now that i use dict of scalar field data array